pkg/controllers/vscr/vdbverify_reconciler.go

```diff
@@ -0,0 +1,81 @@
+/*
+ (c) Copyright [2021-2023] Open Text.
+ Licensed under the Apache License, Version 2.0 (the "License");
+ You may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+*/
+
+package vscr
+
+import (
+	"context"
+
+	"github.com/go-logr/logr"
+	v1 "github.com/vertica/vertica-kubernetes/api/v1"
+	vapi "github.com/vertica/vertica-kubernetes/api/v1beta1"
+	"github.com/vertica/vertica-kubernetes/pkg/controllers"
+	verrors "github.com/vertica/vertica-kubernetes/pkg/errors"
+	"github.com/vertica/vertica-kubernetes/pkg/events"
+	"github.com/vertica/vertica-kubernetes/pkg/version"
+	"github.com/vertica/vertica-kubernetes/pkg/vk8s"
+	corev1 "k8s.io/api/core/v1"
+	ctrl "sigs.k8s.io/controller-runtime"
+)
+
+// VDBVerifyReconciler will verify the VerticaDB in the Vscr CR exists
+type VDBVerifyReconciler struct {
+	VRec *VerticaScrutinizeReconciler
+	Vscr *vapi.VerticaScrutinize
+	Vdb  *v1.VerticaDB
+	Log  logr.Logger
+	VInf *version.Info
```

Comment:
If we want to make a copy of the version, we can just change this to not be a pointer.
```suggestion
	VInf version.Info
```

Reply:
the thing is it is used by the next reconciler that is why it is a pointer.

Reply:
Okay, but we don't need a Copy function. Or maybe make the Copy function just do Go lang struct copy.
```
func (v *Info) Copy() Info {
  return *v
}
```

Can we have a comment too about why you need to make the copy. It wasn't obvious when looking at the code.